maintenance/wikia/importMessagingWiki.php

```diff
@@ -0,0 +1,185 @@
+<?php
+require_once __DIR__ . '/../Maintenance.php';
+
+class ImportMessagingWiki extends Maintenance {
+
+	/** @var string MESSAGING_WIKI_DB messaging wiki database name */
+	const MESSAGING_WIKI_DB = 'messaging';
+
+	/**
+	 * @var array Array of titles to delete (strings)
+	 */
+	protected $articlesToDelete = [];
```

Comment:
I think we can just get rid of this variable and its uses - corresponding logic is not implemented and it's not even needed since we'll just close down messaging wiki for good. Not sure why I added this in the first place.